src/java/voldemort/store/socket/clientrequest/ClientRequestExecutor.java

```diff
@@ -80,6 +80,12 @@ public ClientRequestExecutor(Selector selector,
 
     }
 
+    @Override
+    protected String getDebugInfo() {
+        return "Destination: " + String.valueOf(socketDesination) + " . Socket: "
```

Comment:
I love this technique for printing more useful stuff in error messages, but the period "." is going to look odd when formatted in some of the messages in SelectorManagerWorker.java